docs/cedarling/getting-started/java.md

```diff
@@ -0,0 +1,206 @@
+---
+tags:
+  - cedarling
+  - java
+  - getting-started
+---
+
+# Getting Started with Cedarling Java
+
+- [Installation](#installation)
+- [Usage](#usage)
+
+## Installation 
+
+## Building from Source
+
+Refer following [guide](../uniffi/cedarling-kotlin.md#building-from-source) for steps to build the Java binding from source.
+
+## Using Cedarling-java Maven dependency
+
+### Prerequisites
+
+- Java Development Kit (JDK): version 11 or higher
+
+To use Cedarling Java bindings in Java Maven Project add following `repository` and `dependency` in pom.xml of the project
+
+```declarative
+    <repositories>
+        <repository>
+            <id>jans</id>
+            <name>Janssen project repository</name>
+            <url>https://maven.jans.io/maven</url>
+        </repository>
+    </repositories>
+```
+```declarative
+        <dependency>
+            <groupId>io.jans</groupId>
+            <artifactId>cedarling-java</artifactId>
+            <version>{latest-jans-stable-version}</version>
+        </dependency>
+```
+
+## Usage
+
+### Initialization
+
+We need to initialize Cedarling first.
+
+```java
+
+        import uniffi.cedarling_uniffi.*;
+
+        String bootstrapJsonStr = """
+            {
+            "CEDARLING_APPLICATION_NAME":   "MyApp",
+            "CEDARLING_POLICY_STORE_ID":    "your-policy-store-id",
+            "CEDARLING_USER_AUTHZ":         "enabled",
+            "CEDARLING_WORKLOAD_AUTHZ":     "enabled",
+            "CEDARLING_LOG_LEVEL":          "INFO",
+            "CEDARLING_LOG_TYPE":           "std_out",
+            "CEDARLING_POLICY_STORE_LOCAL_FN": "/path/to/policy-store.json"
+        }
+        """;
+        
+        try {
+            CedarlingAdapter cedarlingAdapter = new CedarlingAdapter();
+            cedarlingAdapter.loadFromJson(bootstrapJsonStr);
+        } catch (CedarlingException e) {
+            log.error("Unable to initialize Cedarling" + e.getMessage());
```

Comment:
What logger are we using here? Maybe just use `println()` if it doesn't matter.

Reply:
Now using `System.out.println`.